http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=455619
## 10_fix_gcc-4.3_ftbfs.dpatch by Cyril Brulebois <cyril.brulebois@enst-bretagne.fr>

--- src/apop.cc
+++ src/apop.cc
@@ -28,6 +28,8 @@
 #include "md5.h"
 }
 
+#include <cstring>
+
 using namespace std;
 
 bool APOP :: login (const char* usr, 
